<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookslist</class>
 <widget class="QDialog" name="bookslist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibTech</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Pictures.qrc">
    <normaloff>:/new/prefix1/pics/LibTechLogo.png</normaloff>:/new/prefix1/pics/LibTechLogo.png</iconset>
  </property>
  <widget class="QLabel" name="coverLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(109, 4, 4);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookNameLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="genreLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>450</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantityLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="publisherLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>531</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="publishedDateLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>610</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="booksTable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(109, 4, 4);
color: rgb(255, 255, 255);
border: none
</string>
   </property>
   <property name="text">
    <string>Delete </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>701</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Pictures.qrc">:/new/prefix1/pics/booklist2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(109, 4, 4);
color: rgb(255, 255, 255);
border: none
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sortComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none</string>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
  </widget>
  <zorder>label</zorder>
  <zorder>coverLabel</zorder>
  <zorder>idLabel</zorder>
  <zorder>bookNameLabel</zorder>
  <zorder>authorLabel</zorder>
  <zorder>genreLabel</zorder>
  <zorder>quantityLabel</zorder>
  <zorder>publisherLabel</zorder>
  <zorder>publishedDateLabel</zorder>
  <zorder>descriptionLabel</zorder>
  <zorder>booksTable</zorder>
  <zorder>searchLineEdit</zorder>
  <zorder>deleteButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>sortComboBox</zorder>
 </widget>
 <resources>
  <include location="Pictures.qrc"/>
 </resources>
 <connections/>
</ui>
